.github/workflows/release-nmtc.yml

```diff
@@ -15,12 +12,12 @@ jobs:
     steps:
       - id: create
         name: Create staging repository
-        uses: nexus-actions/create-nexus-staging-repo@v1
+        uses: nexus-actions/create-nexus-staging-repo@67d9c1133c2fdcb777e20c040595073e20311225
```

Comment:
Why not a stable version?

Reply:
I am ready to rely on well-known actions maintained by teams, but this one unfortunately is not and we are passing very critical information there.

Also it is [advice](https://github.com/nexus-actions/create-nexus-staging-repo) from the author:

```       
# replace '@main' with the latest commit sha, see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
```

Reply:
TIL